tasks/main.yml

```diff
@@ -156,6 +156,130 @@
               name: sosedoff/pgweb:0.11.3
             name: latest
 
+  - name: "create pgbouncer deployment"
+    k8s:
+      state: present
+      definition:
+        apiVersion: v1
```

Comment:
I think this is not the correct api version?!

Reply:
Correct, adjusted also for other objects.